File: prime.py
#!/usr/bin/python3

def prime(num):
    for i in range(2, int(num / 2)+1):
        if num % i == 0:
            return False
    return True

def main():
    num = 0;
    for i in range(2, 100000+1):
        if prime(i):
            num = num + 1
    print("number of primes = %d" % (num))

main()

File: prime.js
#!/usr/bin/node

function prime(num) {
    for (var i = 2; i <= num/2; i++)
        if (num % i == 0)
            return false;
    return true;
}

function main() {
    var num = 0;
    for (var i = 2; i <= 100000; i++)
        if (prime(i))
            num++;
    console.log("number of primes = %d", num);
}

main();
